spec.bs

```diff
@@ -116,6 +116,15 @@ Take the handwritten text "int" for example:
   If the application wants to delete character "t", it can remove stroke 3 and 4 from the drawing.
 </div>
 
+<h2 id="api-idioms">API Idioms</h2>
+
+The [=task source=] mentioned in this specification is the <dfn data-export id="handwriting-recognition-task-source">handwriting recognition task source</dfn>.
+
+When an algorithm <dfn data-export id="queue-task">queues a Handwriting Recognition API task</dfn> |T|, the user agent *MUST* [=queue a global task=] |T| on the [=handwriting recognition task source=] using the [=global object=] of the [[ecma-262#current-realm|current realm]].
```

Comment:
A few suggestions:
- The algorithm typically gets called from algorithm steps using an imperative form. Create an alternative name to avoid the "s".
- Bikeshed does not really know which definition of "global object" to use, make it clearer that you're using the main one in the HTML spec.
- Bikeshed can alert when a term defined in another spec no longer exists. It does not alert about broken fragments though. To avoid making the link explicit, use the actual term "the current realm record" defined in ECMAScript (we didn't do it in the Presentation API, but we should have). That should also play nicer with the index of terms generated at the end of the specification.
- Bikeshed takes care of IDs on its own. Unless there's a specific reason to force an ID, it's usually a good idea to leave that up to Bikeshed.

```suggestion
When an algorithm <dfn data-export lt="queues a Handwriting Recognition API task">queues a Handwriting Recognition API task</dfn> |T|, the user agent *MUST* [=queue a global task=] |T| on the [=handwriting recognition task source=] using the [=/global object=] of [=the current realm record=].
```